Gruntfile.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    uncss: {
+      dist: {
+        options: {
+          stylesheets: [
+            'assets/css/styles.css'
+          ]
+        }
+        files: {
+          'tidy.css': [
+            'http://127.0.0.1:4000/',
+            'http://127.0.0.1:4000/team/',
+            'http://127.0.0.1:4000/team/jamie/',
+            'http://127.0.0.1:4000/consulting/',
+            'http://127.0.0.1:4000/blog/',
+            'http://127.0.0.1:4000/2015/08/10/18f-design-methods/',
+            'http://127.0.0.1:4000/tags/open-source/',
+            'http://127.0.0.1:4000/linking-policy/'
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.